README.md

```diff
@@ -115,38 +120,44 @@ You can see collection of all configuration files at *$TIZENRT_BASEDIR/build/con
 To check all pre-defined configurations, type as follows:
 
 ```bash
-./configure.sh --help
+./tools/configure.sh --help
 ```
 
-##### 1.1 Additional Configuration (optional)
-
 After basic configuration by [1. Configuration](#1-configuration), you can additionally modify your configuration with *menuconfig*.
 
 ```bash
-./dbuild.sh menuconfig
+make menuconfig
 ```
 
-**Note**: In Docker environment, ```make menuconfig``` command from other README files should be replaced with this command.  
-          ```make menuconfig``` applies only in [Manual Setup Build Environment](docs/HowToSetEnv.md).
-
 #### 2. Compilation
-
+There are two aspects to compilation, namely Build and Clean, which are described as follows:
+##### Build
+To compile, simply type the following:
 ```bash
 ./dbuild.sh
 ```
+Additionally, you can modify your configuration with *menuconfig*.
+
+```bash
+./dbuild.sh menuconfig
+```
+
+>**Note**
+>In Docker environment, ```make menuconfig``` command from other README files should be replaced with this command.
+>```make menuconfig``` applies only in [Manual Setup Build Environment](docs/HowToSetEnv.md).
```

Comment:
line 139~147 is mis-added. This is for 1.1, not for 2

Reply:
Modified as suggested. Thanks.